orchagent/vnetorch.h

```diff
@@ -22,6 +23,11 @@
 #define VXLAN_ENCAP_TTL 128
 #define VNET_BITMAP_RIF_MTU 9100
 
+#define VNET_TUNNEL_TERM_ACL_TABLE_TYPE "VNET_LOCAL_ENDPOINT_REDIRECT"
+#define VNET_TUNNEL_TERM_ACL_TABLE "VNET_LOCAL_ENDPOINT"
+#define VNET_TUNNEL_TERM_ACL_BASE_PRIORITY 9998
+#define VNET_TUNNEL_TERM_ACL_RULE_NAME_SUFFIX "IN_TUNN_TERM"
```

Comment:
Since you are matching on DST_IP and it is direction agnostic. I think it is clear to name this "_TUNN_TERM"